### PR TITLE
improve CI actions and refactor package names

### PR DESCRIPTION
**Pull request by qguyk**

### CODE REVIEW

File: qualang_tools/bakery/xeb.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass
-from typing import List, Callable, Union
```
Comment (yomach) [inline qualang_tools/bakery/xeb.py:2]: Why are these removed? They are not used?
If I remember correctly, @arthurostrauss added them because of an issue Caltech had...

Comment (arthurostrauss) [reply qualang_tools/bakery/xeb.py:2]: Correct, those aliases worked and were used to assert the type of the expected arguments in all the methods. Caltech had the issue when the type was directly specified (list instead of List for example).

Comment (qguyk) [reply qualang_tools/bakery/xeb.py:2]: that makes sense, but they are not used in this file. I've added linter so deleted the unused imports

Comment (yomach) [reply qualang_tools/bakery/xeb.py:2]: I think that they were used in the type validation, not?

Comment (yomach) [reply qualang_tools/bakery/xeb.py:2]: It is used in the bakery file, it is unneeded here.

**Review by yomach — APPROVED** [review]
